tests/test_management_commands.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from sys import version
+
+if version[0] == '2':
+    import mock
+else:
+    from unittest import mock
+
+from django.conf import settings
+from pytest import raises
+
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_jwt.management.commands.obtain_token import Command
+
+
+def test_obtain_token_command_should_fail_in_DEBUG_False(
+        monkeypatch, user
+):
+    stdout = mock.Mock()
+    command = Command()
+    monkeypatch.setattr(command, "stdout", stdout)
```

Comment:
Instead of mocking it, you could use `StringIO`. You can also pass arguments directly to the `call_command` function, eliminating the need for the command instantiation and individual method class so the tests serves as a form of documentation:

```python
from django.utils.six import StringIO

def test(...):
    output = StringIO()
    call_command('obtain_token', str(user.pk) stdout=output)
```

Reply:
I didn't know about these, thanks :+1: